src/build-instructions.md

```diff
@@ -83,6 +83,39 @@ libXrandr-devel giflib-devel pulseaudio-libs-devel libxkbfile-devel \
 openssl-devel llvm libcap-devel libavcodec-free-devel libavformat-free-devel
 ```
 
+**Rocky Linux 9**
+
+Enable the required repositories:
+
+```bash
+# Enable RPM Fusion repositories
+sudo dnf install https://download1.rpmfusion.org/free/el/rpmfusion-free-release-9.noarch.rpm
+sudo dnf install https://download1.rpmfusion.org/nonfree/el/rpmfusion-nonfree-release-9.noarch.rpm
+
+# Update package list
+sudo dnf update
```

Comment:
IMO, it's better to direct users to the official guide.

Such as this one: https://rpmfusion.org/Configuration